test/service/ConfigLoader.test.ts

```diff
@@ -43,32 +62,153 @@ describe('ConfigLoader', () => {
                 customPresetObject: undefined,
                 password: 'abc',
             });
+            expect(false).to.be.eq(true);
         } catch (e) {
-            expect(e.message).to.equal(
-                'Preset testnet requires assembly (-a, --assembly option). Possible values are: api, dual, peer. Please provide the --assembly parameter when running the config/start command.',
+            expect(e.message).eq(
+                'Preset testnet requires assembly (-a, --assembly option). Possible values are: api, demo, dual, multinode, peer',
             );
-            return;
         }
-        expect(true).to.be.false;
+    });
+
+    it('ConfigLoader loadPresetData bootstrap no assembly', async () => {
+        const configLoader = new ConfigLoaderMocked(logger);
+        const presetData = await configLoader.createPresetData({
+            preset: Preset.bootstrap,
+            customPreset: undefined,
+            customPresetObject: undefined,
+            password: 'abc',
+        });
+        expect(presetData).to.not.be.undefined;
+        expect(presetData.preset).to.eq(Preset.bootstrap);
+        expect(presetData.assembly).to.eq(Assembly.multinode);
     });
 
     it('ConfigLoader loadPresetData testnet assembly', async () => {
         const configLoader = new ConfigLoaderMocked(logger);
         const presetData = await configLoader.createPresetData({
             preset: Preset.testnet,
-            assembly: 'dual',
+            assembly: Assembly.api,
             customPreset: undefined,
             customPresetObject: undefined,
             password: 'abc',
         });
         expect(presetData).to.not.be.undefined;
+        expect(presetData.preset).to.eq(Preset.testnet);
+        expect(presetData.assembly).to.eq(Assembly.api);
+    });
+
+    it('ConfigLoader custom maxUnlockedAccounts', async () => {
+        const configLoader = new ConfigLoaderMocked(logger);
+        const originalPresetData = await configLoader.createPresetData({
+            preset: Preset.testnet,
+            assembly: Assembly.dual,
+            customPreset: 'test/unit-test-profiles/custom_preset.yml',
+            customPresetObject: {
+                maxUnlockedAccounts: 30,
+            },
+            password: 'abcd',
+        });
+        expect(originalPresetData).to.not.be.undefined;
+        expect(originalPresetData.nodes![0].maxUnlockedAccounts).eq(undefined);
```

Comment:
are you expecting originalPresetData.nodes to be undefined or originalPresetData.nodes![0].maxUnlockedAccounts ? since this is a test, you can be more explicit.

Reply:
I have removed `originalPresetData.nodes![0].maxUnlockedAccounts` assertions. It didn't make any sense.